python/datasources/acs_poverty.py

```diff
@@ -153,6 +215,33 @@ def getData(self, gcs_bucket=None):
                     )
                 self.accumulate_acs_data(data)
 
+    def custom_accumulations(self):
+        new_data = {}
+
+        for data, population in self.data.items():
+            state_fip, county_fip, age, sex, race = data
+
+            above = int(population[PovertyPopulation.ABOVE])
+            below = int(population[PovertyPopulation.BELOW])
+
+            assert above != -1
```

Comment:
I don't really like the idea of having `assert` statements in our code. I would prefer checking the value and `raising` an error if it is wrong.